ocfweb/docs/docs/services/hpc.md

```diff
@@ -92,22 +96,29 @@ Linux on your system, you can install Singularity from the official apt repos:
 sudo apt install singularity-container
 ```
 
-If you are running Mac you can look [here][mac_install], or Windows [here][win_install].
+If you do not have an apt-based Linux distribution, installation instructions
+can be found [here][linux_install]. Otherwise, if you are running Mac you can
+look [here][mac_install], or Windows [here][win_install].
 
 #### Building Your Container
 
 ```
-singularity build --sandbox ./my_container docker://ubuntu
+singularity build [--sandbox] ./my_container docker://ubuntu
 ```
 This will create a Singularity container named `my_container`. If you are still
 testing out your container on your system, we suggest you use the `--sandbox`
 option. This will allow you to install new packages in your container as you
```

Comment:
Is there a downside to using `--sandbox`? If not, I would suggest we just tell them to use it anyway. It would not be helpful to confuse people with unnecessarily complex instructions.

Reply:
Well the downside is that `--sandbox` is not ideal for a finished container because its dealing with a directory rather than a single file. But since this is for when you're building a container then it may be fine to just tell them to use it anyway? @redplanks may have some more opinions on this.